README.md

```diff
@@ -664,7 +688,8 @@ $ helm upgrade -i nvdp nvdp/nvidia-device-plugin \
 For multiple config files, the procedure is similar.
 
 Create a second `config` file with the following contents:
-```
+
+```yaml
```

Comment:
Only part of this block is yaml. Strictly speaking this is `shell`.

Reply:
Yes, but the majority of the contents are YAML and that syntax highlights the contents the best, but happy to change if desired.

Reply:
Hey @chipzoller , thanks for the PR,  Please change it to yaml

Reply:
It is currently set to yaml. Did you mean shell?

Reply:
Ups yeah shell

Reply:
Done